readthedocs/proxito/tests/test_full.py

```diff
@@ -35,6 +35,13 @@
 class TestFullDocServing(BaseDocServing):
     # Test the full range of possible doc URL's
 
+    def test_health_check(self):
+        url = '/_health_check/'
+        host = 'project.dev.readthedocs.io'
+        resp = self.client.get(url, HTTP_HOST=host)
+        self.assertEqual(resp.status_code, 200)
+        self.assertEqual(resp.json, {'ok'})
```

Comment:
Will probably need to update this too. I don't think encoding above will work though, given it's a set.